src/APIInterfaces/BackendAPI/ResponseTypes/UserResponseTypes.ts

```diff
@@ -22,4 +22,7 @@ export type GetUserResponse = AxiosResponse<GetUser>;
 type RegisterUser = {
     verificationBypass: boolean;
 };
-export type RegisterUserResponse = AxiosResponse<BackendAPIResponse<RegisterUser>>;
\ No newline at end of file
+export type RegisterUserResponse = AxiosResponse<BackendAPIResponse<RegisterUser>>;
+
+type GetVerification = unknown;
```

Comment:
Why is this `unknown`? If we're not sending anything back, shouldn't it be `never`?

Reply:
It returns something we just don't use it so I didn't type it
I don't know what advantage never would have